modules/service/src/main/scala/lucuma/odb/service/ObservationWorkflowService.scala

```diff
@@ -352,7 +354,7 @@ object ObservationWorkflowService {
               case Inactive   => List(executionState.getOrElse(validationStatus))
               case Undefined  => List(Inactive)
               case Unapproved => List(Inactive)
-              case Defined    => List(Inactive) ++ Option.when(isAccepted)(Ready)
+              case Defined    => List(Inactive) ++ Option.when(isAccepted || info.tpe === ProgramType.Engineering)(Ready)
```

Comment:
What's true for `Engineering` is probably true for `Calibration` and all the others?  `info.tpe =!= ProgramType.Science`

Reply:
That is sensible, I'll make that change.